AR1/reduction/interferometric_pointing/analyse_interferometric_pointing.py

```diff
@@ -161,8 +162,10 @@ def _eq_TGT_(tgt): # tgt==TGT, "tags" don't matter
         pol_ind['VV'] = np.arange(1.0*chunk_size,2.0*chunk_size,dtype=int) 
         pol_ind['I']  = np.arange(0.0*chunk_size,2.0*chunk_size,dtype=int) 
         for ant in range(len(h5.ants)):
-            h_pol = ~np.isnan(gaussian_centre[pol_ind['HH'],:,ant]) & ~np.isnan(1./gaussian_centre_std[pol_ind['HH'],:,ant])
-            v_pol = ~np.isnan(gaussian_centre[pol_ind['VV'],:,ant]) & ~np.isnan(1./gaussian_centre_std[pol_ind['VV'],:,ant])
+            h_height = np.concatenate((gaussian_height[pol_ind['HH'],ant,np.newaxis] > 0.01, gaussian_height[pol_ind['HH'],ant,np.newaxis] > 0.01), axis=-1) 
```

Comment:
This is because the other checks of valid positions have two coordinates (Az & el)
e.g.
`h_pol = ~np.isnan(gaussian_centre[pol_ind['HH'],:,ant]) & ~np.isnan(1./gaussian_centre_std[pol_ind['HH'],:,ant]) & h_height`

Yes it is a hack 
I will use np.stack